apps/golden-sample-app/src/app/app.component.html

```diff
@@ -47,128 +47,52 @@
-        *bbIfEntitlements="triplets.canViewTransactions"
-      >
-        <a
-          class="bb-layout__horizontal-nav-link"
-          routerLink="/transactions"
-          routerLinkActive="bb-layout__horizontal-nav-link--active"
-          #transactionsListRouterLink="routerLinkActive"
-          [attr.aria-current]="
-            transactionsListRouterLink.isActive ? 'page' : undefined
-          "
-        >
-          <bb-icon-ui
-            class="bb-layout__vertical-nav-item-icon"
-            name="transactions"
-            size="md"
-          ></bb-icon-ui>
-          <span
-            class="bb-layout__horizontal-nav-item-description"
-            i18n="transactions list link@@main.transactions.link.text"
-          >
-            Transactions List
-          </span>
-        </a>
-      </li>
-      <li
-        class="bb-layout__horizontal-nav-item"
-        *bbIfEntitlements="triplets.canViewPositivePay"
-      >
-        <a
-          class="bb-layout__horizontal-nav-link"
-          routerLink="/positive-pay"
-          routerLinkActive="bb-layout__horizontal-nav-link--active"
-          #positivePayRouterLink="routerLinkActive"
-          [attr.aria-current]="
-            positivePayRouterLink.isActive ? 'page' : undefined
-          "
-        >
-          <bb-icon-ui
-            class="bb-layout__vertical-nav-item-icon"
-            name="note"
-          ></bb-icon-ui>
-          <span
-            class="bb-layout__horizontal-nav-item-description"
-            i18n="Positive pay link@@main.positive-pay.link.text"
-          >
-            Positive Pay
-          </span>
-        </a>
-      </li>
+      @for (journey of journeys; track journey) { @if(journey.permissions) {
       <li
+        *bbIfEntitlements="journey.permissions"
         class="bb-layout__horizontal-nav-item"
-        *bbIfEntitlements="triplets.canViewAchRule"
       >
         <a
           class="bb-layout__horizontal-nav-link"
-          routerLink="/ach-positive-pay"
+          [routerLink]="journey.route"
           routerLinkActive="bb-layout__horizontal-nav-link--active"
-          #myAccountsRouterLink="routerLinkActive"
-          [attr.aria-current]="
-            myAccountsRouterLink.isActive ? 'page' : undefined
-          "
+          #routerLink="routerLinkActive"
+          [attr.aria-current]="routerLink.isActive ? 'page' : undefined"
         >
+          @if (journey.icon) {
           <bb-icon-ui
             class="bb-layout__vertical-nav-item-icon"
-            name="verified-user"
+            [name]="journey.icon"
           ></bb-icon-ui>
+          }
 
-          <span
-            class="bb-layout__horizontal-nav-item-description"
-            i18n="Ach Positive pay link@@main.ach-positive-pay.link.text"
-          >
-            ACH Positive Pay
+          <span class="bb-layout__horizontal-nav-item-description">
+            {{ journey.name }}
           </span>
         </a>
       </li>
+      } @else {
```

Comment:
There's a lot of duplication of html structure here, just because `*bbIfEntitlements` will display nothing when given undefined.  Maybe we can avoid this using an ng-template...

```html
      @for (journey of journeys; track journey) {

        <ng-template #journeyLinkTemplate>
          <li class="bb-layout__horizontal-nav-item">
             <!-- ...etc -->
          </li>
        </ng-template>

        @if(journey.permissions) {
          <ng-container *bbIfEntitlements="journey.permissions">
            <ng-container *ngTemplateOutlet="journeyLinkTemplate" />
          </ng-container>
        } @else {
          <ng-container *ngTemplateOutlet="journeyLinkTemplate" />
        }
```

Maybe even better would be to speak with the web-foundation team to ask if they can change the `*bbIfEntitlements` directive so that if given a certain value, say `'*'`, then it automatically shows the content without checking any entitlements.  Then we could just do this:

```html
      <li
        *bbIfEntitlements="journey.permissions ?? '*'"
        class="bb-layout__horizontal-nav-item"
      >
        <!-- ... etc -->
      </li>
```